pre_commit/languages/lua.py

```diff
@@ -0,0 +1,113 @@
+import contextlib
```

Comment:
I tend to order functions in the opposite order of what you've written here -- where the helpers are at the top and the api is at the bottom and it calls things above it

Reply:
I'm a "most abstract thing first" kind of guy, but I have no problem switching the order to your preference. 😄